Gemfile

```diff
@@ -10,6 +10,7 @@ gem 'em-http-request',                  :require => false, :platforms => [:ruby,
 gem 'em-synchrony',                     :require => false, :platforms => [:ruby, :jruby]
 gem 'excon',               '~> 0.21',   :require => false, :platforms => [:ruby, :jruby]
 gem 'net-http-persistent', '~> 2.8',    :require => false
+gem 'multipart-post',                   :require => false
```

Comment:
Hi @keeguon. Looking again all changes, i realized that is missing the second argument and maybe this is a problem to jruby. You can try to change to:

`gem 'multipart-post', '~> 2.0', :require => false, :platforms => [:ruby, :jruby]`